osx/qTox-Mac-Deployer-ULTIMATE.sh

```diff
@@ -245,52 +248,64 @@ function bootstrap() {
 	sudo ./bootstrap-osx.sh
 }
 
-# The commands
-if [[ "$1" == "-i" ]]; then
+function dmgmake() {
+	fcho "------------------------------"
+	fcho "Starting DMG creation"
```

Comment:
Said this in IRC but for context `fcho` is a custom function used in various points throughout the script. https://github.com/tux3/qTox/blob/master/osx/qTox-Mac-Deployer-ULTIMATE.sh#L53